pkg/registry/generic/rest/streamer.go

```diff
@@ -56,6 +62,13 @@ func (s *LocationStreamer) InputStream(apiVersion, acceptHeader string) (stream
 	if err != nil {
 		return
```

Comment:
This method is complex enough that explicit return statement would be better (should have raised it in the original code review) - `return nil, false, "", err`